packages/application-shell/src/configure-store.js

```diff
@@ -50,32 +50,34 @@ export const createReduxStore = (
       notifications: notificationsReducer,
       ...injectedReducers,
     });
+  const middlewares = [
+    // Should be defined before `createExtractGlobalActions`
+    addPluginToNotificationMiddleware,
+    createExtractGlobalActions([
+      /* list of action types plugins may dispatch globally */
+      SHOW_LOADING,
+      HIDE_LOADING,
+      ADD_NOTIFICATION,
+      REMOVE_NOTIFICATION,
+      HIDE_ALL_PAGE_NOTIFICATIONS,
+    ]),
+    createScopedMiddleware(thunk),
+    sentryTrackingMiddleware,
+    createScopedMiddleware(sentryTrackingMiddleware),
+    hideNotificationsMiddleware,
+    notificationsMiddleware,
+    sdkMiddleware,
+    createScopedMiddleware(sdkMiddleware),
+    thunk,
+    batchedUpdates,
+  ];
+  if (process.env.NODE_ENV !== 'test') middlewares.push(loggerMiddleware);
```

Comment:
I already took care of it in https://github.com/commercetools/merchant-center-application-kit/pull/175